### DISCUSSION TITLE
Font Family and Font Style don't work in Linux.

### DISCUSSION THREAD

**Discussion by 23WilliamDing** [QuestPDF/QuestPDF#560] [Q&A]
@MarcinZiabek , your QuestPDF is supper cool. But when I use QuestPDF to render PDF, I meet a issue, maybe need your help. The issue is that QuestPDF works as expected in Windows, but when I deploy it (.net core 3.1 web api application) in the Linux by docker, PDF files doesn't show texts as expected. I know that Linux doesn't support some fonts. So I Register Fonts, but it still doesn't work, do I miss something?  

**What I used in my codes:**
.FontFamily(Fonts.Arial) and .FontFamily(Fonts.CourierNew)
.Bold() and .Italic()

var fontArialPath = Path.Combine(@"\Fonts\ARIAL.TTF");
var fontCourierNewPath = Path.Combine(@"\Fonts\Courier-New.TTF");
QuestPDF.Drawing.FontManager.RegisterFont(File.OpenRead(fontArialPath));
QuestPDF.Drawing.FontManager.RegisterFont(File.OpenRead(fontCourierNewPath));

**Comment by 23WilliamDing** [comment]
What I want:
![image](https://github.com/QuestPDF/QuestPDF/assets/30248071/1997f1d9-ce36-4993-ae28-931002592ba7)
but actual pdf:
![image](https://github.com/QuestPDF/QuestPDF/assets/30248071/70238756-6f72-4f89-b54c-cada45803d74)

**Comment by 23WilliamDing** [comment]
Already fixed, use   **QuestPDF.Drawing.FontManager.RegisterFontWithCustomName("ArialBD",File.OpenRead(fontArialBDPath));**

**Reply by martijndormans** [reply]
This does fix the problem indeed.
